web/src/i18n.js

```diff
@@ -0,0 +1,107 @@
+import LanguageDetector from 'i18next-browser-languagedetector';
+
+i18n
+  // .use(Backend)
+  // detect user language
+  // learn more: https://github.com/i18next/i18next-browser-languageDetector
+  .use(LanguageDetector)
+  // pass the i18n instance to react-i18next.
+  .use(initReactI18next)
+  // init i18next
+  // for all options read: https://www.i18next.com/overview/configuration-options
+  .init({
+    debug: true,
+    fallbackLng: 'en',
+    interpolation: {
+      escapeValue: false, // not needed for react as it escapes by default
+    },
+    resources: {
+      en: {
+        translation: {
+          // here we will place our translations...
+          header: {
+            docs_link: 'API Docs'
+          },
+          jobs: {
+            latest_tab: 'LATEST RUN',
+            history_tab: 'RUN HISTORY',
+            location: 'LOCATION',
+            empty_title: 'No Run Information Available',
+            empty_body: 'Try adding some runs for this job.'
+          },
+          search: {
+            search: 'Search',
+            jobs: 'Jobs',
+            and: 'and',
+            datasets: 'Datasets'
+          }
+        }
+      },
+      fr: {
+        translation: {
+          header: {
+            docs_link: 'API Docs'
+          },
+          jobs: {
+            latest_tab: 'DERNIÈRE COURSE',
+            history_tab: "HISTORIQUE D'EXECUTION",
+            location: 'EMPLACEMENT',
+            empty_title: 'Pas de Course les Informations Disponibles',
+            empty_body: "Essayez d'ajouter quelques exécutions pour ce travail."
+          },
+          search: {
+            search: 'Recherche',
+            jobs: "d'Emplois",
+            and: 'et',
+            datasets: "d'Ensembles de Données"
+          }
+        }
+      },
+      es: {
+        translation: {
+          header: {
+            docs_link: 'API Docs'
+          },
+          jobs: {
+            latest_tab: 'ÚLTIMA EJECUCIÓN',
+            history_tab: 'HISTORIAL DE EJECUCIONES',
+            location: 'UBICACIÓN',
+            empty_title: 'No hay Información de Ejecución Disponible',
+            empty_body: 'Intente agregar algunas ejecuciones para este trabajo.'
+          },
+          search: {
+            search: 'Buscar',
+            jobs: 'Trabajos',
+            and: 'y',
+            datasets: 'Conjuntos de Datos'
+          }
+        }
+      },
+      pl: {
+        translation: {
+          header: {
+            docs_link: 'API Dokumenty'
+          },
+          jobs: {
+            latest_tab: 'OSTATNI BIEG',
+            history_tab: 'BIEGAĆ HISTORIA',
+            location: 'LOKALIZACJA',
+            empty_title: 'Brak dostępnych informacji o przebiegu',
+            empty_body: 'Spróbuj dodać kilka przebiegów dla tego zadania.'
+          },
+          search: {
+            search: 'Wyszukaj Oferty',
+            jobs: 'Pracy',
+            and: 'i',
+            datasets: 'Zestawy Danych'
+          }
```

Comment:
I think we need to agree on run, job, dataset translations. IMO

```
RUN: wykonanie
JOB: zadanie
DATASET: zbiór danych
```

I think only `wykonanie` might be controversial, but I don't have better idea. `Bieg` or `przebieg` make sense only as the sport IMO.

then: 
```
docs_link: Dokumentacja API
latest_tab: OSTATNIE WYKONANIE
history tab: HISTORIA WYKONAŃ
location: LOKALIZACJA
empty_title: Brak dostępnych informacji o wykonaniu
empty_body: Spróbuj dodać kilka wykonań dla tego zadania

search: Wyszukiwanie
jobs: Zadania
and: i
datasets: Zbiory Danych
```

Still, translations of "computer concepts" will always sound awkward in Polish for me.

Reply:
thank you, @mobuchowski . @pawel-big-lebowski @JDarDagran @tnazarew WDYT?

Reply:
I'd say `wykonanie` sounds more natural to me. We shouldn't be so strict about translating this one to one.  Job and dataset are just right.

Reply:
Thanks, @mobuchowski and @JDarDagran . I've revised the translations.